src/phoenix/trace/langchain/tracer.py

```diff
@@ -279,6 +265,19 @@ def _retrieval_documents(
     ]
 
 
+def _null_fallback(
+    serialized: Dict[str, Any],
+    messages: List[List[BaseMessage]],
+    *,
+    run_id: UUID,
+    tags: Optional[List[str]] = None,
+    parent_run_id: Optional[UUID] = None,
+    metadata: Optional[Dict[str, Any]] = None,
+    **kwargs: Any,
+) -> None:
```

Comment:
why not make its intent even more clear?
```suggestion
def _null_fallback(*_: Any, **__: Any) -> None:
```

Reply:
I personally don't believe verbosity or repeated signatures are too onerous to maintain in the sense that I think they should be clearly and directly linked (maybe I should even pick a better name for the fallback function). The function as it looks now is what I intend for this to look and feel like (providing an explicit alternate codepath that has exactly the same usable signature), though I understand if you feel differently.

I'm mostly interested in what you think the shortest resolution might be

Reply:
That's fine, I guess I reached with the opposite conclusion from reading the code alone, so a few clarifying comments will resolve this.

Reply:
Sure thing, let me give that a try and if it still feels bad to you I'll move to the generalized signature

Reply:
I guess it would also have to be renamed to something other than `null_fallback`